app/src/org/commcare/CommCareApplication.java

```diff
@@ -246,7 +246,7 @@ public void onCreate() {
         LocalePreferences.saveDeviceLocale(Locale.getDefault());
     }
 
-    private void attachISRGCert() {
+    protected void attachISRGCert() {

```

Comment:
@shubham1g5 had to do this because the Robolectric tests were throwing problems with the OkHTTP manipulation stuff. This just makes it possible to override attaching the cert there since it wont' be necesary.